.github/workflows/rust.yml

```diff
@@ -104,18 +104,17 @@ jobs:
         run: cargo xtask build-client
 
   tests:
-    runs-on: ubuntu-24.04
+    runs-on: windows-2022
     steps:
       - uses: actions/checkout@v4
         with:
           submodules: true
       - uses: dtolnay/rust-toolchain@stable
```

Comment:
The cache should still be used, even with windows, but most importantly I think this is stupid. Linux is a far more sane environment for testing and if you think we should also test windows, then yes, but separately. Especially because this seems like a hack fix for an unrelated issue.

Reply:
I removed cache because it's not the slowest runner anyway. Having one less runner that saves cache will leave more space headroom in our memory quota for the repository.